clients/drcachesim/analyzer_multi.cpp

```diff
@@ -214,7 +216,16 @@ analyzer_multi_t::create_analysis_tool_from_options(const std::string &tool)
         knobs.verbose = op_verbose.get_value();
         knobs.cpu_scheduling = op_cpu_scheduling.get_value();
         knobs.use_physical = op_use_physical.get_value();
-        return tlb_simulator_create(knobs);
+        analysis_tool_t *tlb_simulator = tlb_simulator_create(knobs);
+
+        std::string v2p_file_path =
+            get_aux_file_path(op_v2p_file.get_value(), DRMEMTRACE_V2P_FILENAME);
+        if (!v2p_file_path.empty()) {
+            static_cast<tlb_simulator_t *>(tlb_simulator)
+                ->create_v2p_from_file(v2p_file_path);
```

Comment:
It seems like reading the file should be done in the tlb simulator constructor or init, instead of here?  That would match other tools like the func_view tool which take in an aux file as an argument.

Reply:
I put it in `tlb_simulator_create()` to avoid changing/adding new constructors for `tlb_simulator_t` and `simulator_t`.